### PR TITLE
Add `shield-question` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

File: icons/shield-question.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z" />
+  <line x1="12" x2="12" y1="17" y2="17" />
+  <path d="M9 9c.6-1.6 2.3-2.4 3.9-1.8 1.3.4 2.1 1.5 2.1 2.8 0 2-3.1 3-3.1 3" />
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/shield-question.svg:14]: Well I did think all the question marks need sorting out and matching in a separate PR…

[the remark this answers is not in the document]
Comment (karsa-mistmere) [reply icons/shield-question.svg]: Yeah, @jguddas as also noted below, I already have like a gazillion related drafts: https://www.figma.com/file/xJ8fi6R8GglEK18IZTWgrF/lucide%2Fshapes?node-id=0%3A1&t=BFd5ZxXC2j4HfezH-1

[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/shield-question.svg]: Looks like my Illustrator document… 😅

Untitled

**Comment by danielbayley** [comment]
> there's an ongoing issue about the question marks

> Correlate this with [feathericons/feather#550](https://github.com/feathericons/feather/issues/550) and #119

Those concerns are for a separate PR though, right…?

**Comment by karsa-mistmere** [comment]
Yes, for now let's keep things consistent. ^_^

**Review by ericfennis — APPROVED** [review]
Really nice!

**Pull request merged**